StarArawn/bevy_ecs_tilemap
# Support Bevy v0.12.0

## Summary
This PR makes the necessary changes in order for this library to be bevy v0.12.0 compliant. This closes #488.

## Testing
1. Clone the `bevy_ecs_tilemap` repository as-is.
2. Run all of the examples as described in the `README.md`.
3. Clone my version of `bevy_ecs_tilemap`
4. Run `cargo check` to verify there are no compilation issues.
5. Run all of the examples as described in the `README.md`.
6. Observe there are (hopefully) no differences between the original.
